### PR TITLE
[WFCORE-1334] Allow disabling audit log rotation

### PR DESCRIPTION
**Pull request by TomasHofman**
https://issues.jboss.org/browse/WFCORE-1334

### CODE REVIEW

**Comment by TomasHofman** [comment]
Investigating the failures.

**Comment by TomasHofman** [comment]
retest this please

File: controller/src/main/java/org/jboss/as/controller/audit/FileAuditLogHandler.java [hunk 1]
```diff
@@ -40,14 +40,18 @@
     //Methods on this class will only ever be called from one thread (see class javadoc) so although it looks shared here it is not
     private static final SimpleDateFormat OLD_FILE_FORMATTER = new SimpleDateFormat("yyyy-MM-dd_HHmmss");
 
-    public FileAuditLogHandler(String name, String formatterName, int maxFailureCount, PathManagerService pathManager, String path, String relativeTo) {
+    private boolean rotateAtStartup;
```
Comment (bstansberry) [inline controller/src/main/java/org/jboss/as/controller/audit/FileAuditLogHandler.java]: This should be final.

**Comment by bstansberry** [comment]
Other than the minor thing about the field being final, this looks good. But we'll need to wait for https://github.com/wildfly/wildfly-core/pull/1366 to be merged and then tweak this to reflect that this is a change to version 5 of the xsd, not 4.

**Comment by bstansberry** [comment]
#1366 is merged, so this can be updated to use the correct xsd version.

**Comment by TomasHofman** [comment]
I updated the PR to use version 5.

**Pull request merged**